src/Console/Command/ReadmeCommand.php

```diff
@@ -236,15 +236,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $help = preg_replace("#^\n( +\\$ )#m", "\n.. code-block:: bash\n\n$1", $help);
         $help = preg_replace("#^\n( +<\\?php)#m", "\n.. code-block:: php\n\n$1", $help);
         $help = preg_replace_callback(
-            "#^\s*<\?(\w+).*?\?>#ms",
+            '#^\\s*<\\?(\\w+).*?\\?>#ms',
```

Comment:
so, it was admitted that escaping in single quoted string doesn't make it better, why to escape ?

Reply:
This PR escaped the double-quoted string, then `SingleQuoteFixer` found an even number of backslashes and converted it into a single-quoted string.

Reply:
so, we could use single `'` notation now ?

Reply:
I don't get what you expect from this PR.

If as a final result you would like a single-quoted string without escaped backslashes, than I think SingleQuoteFixer should be in charge to discard (with a dedication option) the superfluous backslashes.

Or maybe you are simply requesting me to manually edit the particular file, don't you?

Reply:
yeah, if we disabled escaping in single quoted strings by default, no need to escape it here because of old config of rule

Reply:
Still I don't get what you want.
Please reply with "Slamdunk please [verb] [object]".

Reply:
sure thing!
@Slamdunk , please consider applying the following patch:
```diff
-'#^\\s*<\\?(\\w+).*?\\?>#ms',
+'#^\s*<\?(\w+).*?\?>#ms',
```
as you configured the fixer to not touch single-quoted strings by default.

Reply:
> please consider applying the following patch

Ok, done manually.

> as you configured the fixer to not touch single-quoted strings by default.

This statement seems to me senseless. A code like `"a\b\c"` with this fixer and SingleQuoteFixer would always result in `'a\\b\\c'`, the `single_quoted` configuration doesn't matter because is not involved.

Reply:
in that case, `single_quote` shall run before `escape_implicit_backslashes`

Reply:
I'm against this. Single quote string is less complex to write and understand, and fixers that work together toward this goal should be encouraged.
I understand though that the result may be verbose: I can propose a new fixer opposite to this one that removes useless backslashes in single quoted strings. That new fixer would run at the end, so we could gain:

1. This fixer straighten backslashes everywhere
1. SingleQuote/HeredocToNowdoc move everything to a less complex string type
1. The new fixer removes verbosity

Reply:
IMO removing extra backslashes could be a new option in `single_quote` rather than a new fixer.

Reply:
```json
{"single_quote": true, "escape_implicit_backslashes": {"single_quoted": false}}
```

current result:
```diff
-"#^\s*<\?(\w+).*?\?>#ms",
+'#^\\s*<\\?(\\w+).*?\\?>#ms',
```

Wow, but we just configured that single_quoted strings shall not have `\` escaped!
With current order, it creates conflict between 2 rules.

----------------

now, imagine that the order is reversed:

```json
{"single_quote": true, "escape_implicit_backslashes": {"single_quoted": false}}
```

new result:```diff
-"#^\s*<\?(\w+).*?\?>#ms",
+'#^\\s*<\\?(\\w+).*?\\?>#ms',
```